fri/server/main.py

```diff
@@ -70,14 +70,17 @@ def build(dir):
     makestudy_dir = dirname + "/" + graphml_file   #for makestudy
     dir_path = os.path.abspath(os.path.join(concore_path, graphml_file)) #path for ./build
     if not os.path.exists(dir_path):
-        proc = call(["./makestudy", makestudy_dir], cwd=concore_path)
+        proc = call(["makestudy", makestudy_dir],shell=True, cwd=concore_path)

```

Comment:
Thanks. So this fix should be repeated for all the methods (clear, run, ...) I believe?

Reply:
yes I will update soon

Reply:
Thanks.